README.md

```diff
@@ -35,4 +37,61 @@ on:
      - 'main'
 ```
 ### Default Behavior
-Using the provided template, without configuring the filter actions, will cause the Shipwright Tokens action to run on any PR opened against `main`.
+Using the provided template, without configuring the filter actions, will cause the Shipwright Tokens action to run on any PR opened against `main`. 
+***
+
+## Integrate Tailwind CSS
+This section will show you how to configure your Shipwright Tokens action to output a custom Tailwind CSS theme.
+
+### Generate a Custom Theme
+Ensure that the `.yml` file that calls Shipwright Tokens includes `tailwind` as the value for the `styleSystem` property:
+```
+    steps:
+      - name: Shipwright Tokens
+        uses: headwayio/shipwright-tokens@(release/version)
+        with:
+          styleSystem: tailwind
+```
+With this configuration option set, Shipwright Tokens will generate 4 files that will be used in your theme: `colors.json`, `misc.json`, `shadows.json`, and `typography.json`. These files will be generated in the directory that you specify for the `outputFolder` property.
+
+### Using the Custom Theme
+With your theme files generated, you will need to configure your `tailwind.config.js` file to use your theme files - [Example Here](./theme_templates/tailwind.config.js)
+
```

Comment:
Can you update the verbiage here with the clarifications we made yesterday about using the theme?

Hightlights:

- There's also a tailwind.config.js file generated, that imports the others.
- It can be used as a preset in your tailwind config (give example of syntax and indicate that this is our suggested way of using these files).
- Offer the current documentation you have here as an alternative for only pulling in specific styles.

Reply:
👍